.tekton/basic-component-pipeline.yaml

```
@@ -0,0 +1,386 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: basic-component
```

After glancing at `main-pipeline.yaml` and `scanner-v4-pipeline.yaml`, I think, it would be better to give more unique names so that searching all code by the name produces fewer and more targeted hits.
I mean renaming
* `basic-component` -> `basic-component-pipeline`
* `main` -> `main-pipeline` (`main` will be especially tricky if you search across all repo)
* `scanner-v4` -> `scanner-v4-pipeline`

```suggestion
  name: basic-component-pipeline
```